### PR TITLE
Updates the Syndicate sniper's description

### PR DESCRIPTION
**Pull request by PopeDaveThe3th**
## What Does This PR Do
title

## Why It's Good For The Game
i forgor this one in the first pr

## Testing
- loaded in
- rifle
- words

## Changelog
:cl:
tweak: updated syndiesniper's desc
/:cl:

### CODE REVIEW

**Review by Henri215 — APPROVED** [review]

**Comment by Fordoxia** [comment]
comment

**Comment by PopeDaveThe3th** [comment]
> comment

reply

**Comment by Henri215** [comment]
> > comment
> 
> reply

reply of the reply?

File: code/modules/projectiles/guns/projectile/sniper.dm [hunk 1]
```diff
@@ -30,8 +30,8 @@
 	return ..()
 
 /obj/item/gun/projectile/automatic/sniper_rifle/syndicate
-	name = "syndicate sniper rifle"
-	desc = "Syndicate flavoured sniper rifle, it packs quite a punch, a punch to your face."
+	name = "\improper SR-31C (S) Sniper Rifle"
+	desc = "A powerful anti-materiel rifle by Aussec Armory, chambered in devastating .50 BMG. This model is engraved with superficial Syndicate iconography."
```
Comment (1080pCat) [inline code/modules/projectiles/guns/projectile/sniper.dm:34]: aussec???
as in, australia?

**Review by 1080pCat — APPROVED** [review]

File: code/modules/projectiles/guns/projectile/sniper.dm (hunk 1 above)
Comment (PopeDaveThe3th) [reply code/modules/projectiles/guns/projectile/sniper.dm:34]: weapon company

Comment (1080pCat) [reply code/modules/projectiles/guns/projectile/sniper.dm:34]: :(

**Review by S34NW — APPROVED** [review]

**Pull request merged**